packages/flutter/test/material/bottom_sheet_test.dart

```diff
@@ -1999,6 +1999,97 @@ void main() {
     });
 
   });
+
+    // ignore: always_declare_return_types
+  group('showModalBottomSheet modalBarrierDismissLabel', () {
```

Comment:
Looks like this group message is same as the below one, and the test is similar with the first test in the group below. Should these tests be put into one group, or maybe this group can just be removed?